.github/workflows/configurable-benchmark-test.yaml

```diff
@@ -35,12 +35,13 @@ jobs:
     if: ${{ github.event_name == 'workflow_dispatch' }}
     steps:
       - name: Setup Kubernetes
-        uses: manusa/actions-setup-minikube@v2.4.3
+        uses: medyagh/setup-minikube@v0.0.7

```

Comment:
You can set cpu and memory like: `start args: "--cpu 4 --memory 5192"` without changing repo.